PWGCF/FEMTOSCOPY/FemtoDream/AliFemtoDreamTrackHist.cxx

```diff
@@ -453,7 +453,7 @@ AliFemtoDreamTrackHist::AliFemtoDreamTrackHist(TString MinimalBooking)
 
   fpTDist[0] = 0;
   TString ptName = Form("pTDist_%s", "after");
-  fpTDist[1] = new TH1F(ptName.Data(), ptName.Data(), 200, 0, 5);
+  fpTDist[1] = new TH1F(ptName.Data(), ptName.Data(), 200, 0, 6.0);
```

Comment:
Change the nBins accordingly so the binwidth stays the same

Reply:
Now I changed bin size accordingly